sys/hashes/sha256.c

```diff
@@ -49,7 +49,6 @@
 #include <assert.h>
 
 #include "hashes/sha256.h"
-#include "hashes/sha2xx_common.h"
```

Comment:
Is this correct? Line 111 in this file uses `sha2xx_update(..)` which is defined in the removed header.

Reply:
Well it's transitively included via `hashes/sha256.h` but probably better to keep the include explicit.